reconcile/test/test_utils_slack_api.py

```diff
@@ -168,6 +175,39 @@ def test_chat_post_message_missing_channel(slack_api):
         slack_api.client.chat_post_message('test')
 
 
+def test_chat_post_message_channel_not_found(mocker, slack_api):
+    slack_api.client.channel = 'test'
+    mock_join = mocker.patch('reconcile.utils.slack_api.SlackApi.join_channel')
```

Comment:
I'd generally use `autospec=True` on all patches unless there is a good reason not to. This might be one of those good reasons not to, but just pointing it out in case it isn't.

Reply:
While we're at it, do we have a `patch.object` for this `mocker` object? It's usually even more reliable than bare `patch`.

Reply:
@Piojo good point, I'll add this to this ticket: APPSRE-4341